### ISSUE TITLE
Can't make any recipes in the alchemical chemistry set

### ISSUE THREAD

**Issue by amburo** [talonos2/BlightfallTracker#188]
Running 2.5 on windows: I was trying to do the diamond sword quest in the Dark One tab and was unable to make one despite having the exact recipe, set the exact way. Also tested other recipes--same issue. Had the proper LP, proper set up, proper blood orb, nothing.

**Comment by Raclex** [comment]
I know this is old, but are you sure the blood orb was bound to you? Need to check its description and make sure you are the owner. If not, right click it in the air.

**Comment by amburo** [comment]
It was bound to me, doubly so considering I was messing around in a single-player world

**Comment by Raclex** [comment]
Hmm...  I had no issue... starting with bottom left slot going clockwise, diamond, stone sword blade, stick, diamond. Your BOUND blood orb in the chemistry set with LP in your soul network, and you aren't getting anything?

"To create an item with the chemistry set, place the ingredients in the five slots surrounding the middle and a Blood Orb in the lower right slot. Some recipes require higher tier orbs than others. The output will be placed into the middle slot. The chemistry set will drain Life Essence from the Soul Network and produce the item. While it is operating, it will emit red particles. To make the blood orb functional, you must right click to drain your life, similar to when using a sacrificial orb and a blood altar."

**Comment by amburo** [comment]
Nope, didnt get anything. I stopped playing a few months ago so it might be fixed at this point, but after a lot of messing around it wouldnt work for me. Figured I'd bring it up even if I no longer continue to play.
